app/Http/Controllers/LocationsController.php

```diff
@@ -266,12 +267,6 @@ public function postRestore($id) : RedirectResponse
             }
 
             if ($location->restore()) {
```

Comment:
The one thing that worries me here i that *some* FCOs use a special `logAndSaveIfBlah()` method, and some have the magic overrides in the `save()`, `saving()`, `restoring()` methods, and I'm just concerned that it could be confusing to figure out where the magic is.